packages/ocaml-webworker/ocaml-webworker.0.0.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "sandermail@gmail.com"
+authors: "Sander Spies"
+homepage: "https://github.com/SanderSpies/ocaml-gist"
+bug-reports: "https://github.com/SanderSpies/ocaml-gist/issues"
+license: "MIT"
+dev-repo: "https://github.com/SanderSpies/ocaml-gist.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
+  "ocamlfind"
+  "js_of_ocaml" {dev}
+  "js_of_ocaml-compiler" {dev}
+  "js_of_ocaml-toplevel"
+  "yojson"
+]
+available: [ocaml-version >= "4.04.1"]
```

Comment:
I think this also needs to be ocaml-version < "4.05.0" since compilation fails with

```
#=== ERROR while installing ocaml-webworker.0.0.1 =============================#
# opam-version         1.2.2 (aa258ecc06d3aea5a67f442a4ffd23f2a457180b)
# os                   linux
# command              jbuilder build -p ocaml-webworker -j 2
# path                 /home/opam/.opam/4.05.0/build/ocaml-webworker.0.0.1
# compiler             4.05.0
# exit-code            1
# env-file             /home/opam/.opam/4.05.0/build/ocaml-webworker.0.0.1/ocaml-webworker-18-942b95.env
# stdout-file          /home/opam/.opam/4.05.0/build/ocaml-webworker.0.0.1/ocaml-webworker-18-942b95.out
# stderr-file          /home/opam/.opam/4.05.0/build/ocaml-webworker.0.0.1/ocaml-webworker-18-942b95.err
### stderr ###
# No rule found for _build/default/ocaml_webworker/merlin_lite/src/ocaml/typer_4.05.0/browse_raw.ml
```

Reply:
Thanks Anil, I already got working on supporting 4.05 but this a faster solution!

Reply:
@SanderSpies are you planning to update the `opam` file in your PR as recommended?

Reply:
@gasche Just did, apologies for the delay.

Reply:
I notice it's still failing though :(